proto/cosmos/feegrant/v1beta1/feegrant.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+package cosmos.feegrant.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/feegrant/types";
+
+// BasicFeeAllowance implements FeeAllowance with a one-time grant of tokens
+// that optionally expires. The delegatee can use up to SpendLimit to cover fees.
+message BasicFeeAllowance {
+  option (cosmos_proto.implements_interface) = "FeeAllowanceI";
+
+  repeated cosmos.base.v1beta1.Coin spend_limit = 1
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+  ExpiresAt expiration = 2 [(gogoproto.nullable) = false];
```

Comment:
if nullable = false, how do we add a no expire allowance?

Reply:
@amaurymartiny That is being handled here, 
https://github.com/cosmos/cosmos-sdk/blob/atheesh/x-fee-grant/x/feegrant/types/expiration.go#L63

Reply:
Has few tests for `ExpiresAt` in `/types/expiration_test.go`, can you verify them.